### PR TITLE
Frontend validation check for empty api key

### PR DESCRIPTION
**Pull request by karans5004**
Add frontend validation for empty API key submission.

Issue #3

### CODE REVIEW

File: app/public/js/api.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
+const emptyApiKeyError = '<p class="red-text red-darken-4">Please Enter API Key</p>'
```
Comment (atiqueahmedziad) [inline app/public/js/api.js:1]: should be a `;` in the end

File: app/public/js/api.js [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+
 $('#api-key-form').submit(e => {
   e.preventDefault();
   const apiKey = $('#api_key').val();
+  if ( $.trim( apiKey ) == '' ){
```
Comment (atiqueahmedziad) [inline app/public/js/api.js:6]: can you please remove the empty spaces?

**Review by atiqueahmedziad — COMMENTED** [review]
Thanks for the pull request. Just 2 nitpicks. Can you please fix these?

File: app/public/js/api.js (hunk 2 above)
Comment (karans5004) [reply app/public/js/api.js]: done.

**Review by atiqueahmedziad — APPROVED** [review]

**Comment by atiqueahmedziad** [comment]
Awesome. Thanks, @karans5004

**Pull request merged**